config/config.exs

```diff
@@ -16,7 +16,9 @@ config :sprinkler, SprinklerWeb.Endpoint,
   secret_key_base: "waMq4QHBtdZppI5ZKJlhHt4MslX/qCH+1Qs1KSS3jACwndYxcOQ226h5yNcGNnl7",
   render_errors: [view: SprinklerWeb.ErrorView, accepts: ~w(html json), layout: false],
   pubsub_server: Sprinkler.PubSub,
-  live_view: [signing_salt: "0rRbKVvU"]
+  live_view: [signing_salt: "0rRbKVvU"],
+  broker_host: "localhost",
```

Comment:
Maybe we can get the URL from the ENV and get localhost as a fallback.
```suggestion
  broker_host: System.get_env("MQTT_BROKER_HOST", "localhost"),
```

We can do the same for the port. Also, this config should not be in the Endpoint. Add it as a different entry